deploy/minikube/schema_check.go

```diff
@@ -45,9 +45,9 @@ func validateSchema(schemaPathString string, docPathString string) {
 	}
 
 	if result.Valid() {
-		fmt.Printf("The document %s is valid\n", docPathString)
+		fmt.Printf("The document %s is valid \n", docPathString)
```

Comment:
-1  In term of programing this was expected. there is no need any space between word and `\n`